lib/facility_factory.rb

```diff
@@ -0,0 +1,30 @@
+class FacilityFactory 
+    def initialize
+        
+    end
+
+    def create_facility(facility_raw_data)
+        facility_raw_data.map do |facility| 
```

Comment:
Well done making it through this method! Adding in new states adds a good deal of complexity really quickly. 

Similar to my last comment above, this becomes a pretty long method so using helper methods would be a cleaner option. Consider creating a helper method for each state and then delegating to that helper method from this top level create_facilities method:

```
def create_facilities(raw_data)
    if raw_data.first[:state] == "CO"
        return create_co_facilities(raw_data)
    elsif raw_data.first[:state] == "MO"
        return create_mo_facilities(raw_data)
    elsif raw_data.first[:state] == "NY" 
        return create_ny_facilities(raw_data)
    else 
       return []
    end
end

def build_co_facilities(raw_data)
    raw_data.map do |facility|
        process facility and return Facility object
    end
end
```